src/e2e-test/java/io/cdap/plugin/utils/PubSubClient.java

```diff
@@ -56,4 +92,76 @@ public static String getTopicCmekKey(String topicId) throws IOException {
     return getTopic(topicId).getKmsKeyName();
   }
 
+
+  public static void publishWithErrorHandlerExample(String projectId, String topicId, List<String> dataMessages)
+      throws IOException, InterruptedException {
+    TopicName topicName = TopicName.of(PluginPropertyUtils.pluginProp(ConstantsUtil.PROJECT_ID), topicId);
```

Comment:
Can we use the projectId Variable from the parameter?

Reply:
this code was already used before and its taking project id from plugin properties inly